website/docs/docs/build/cumulative-metrics.md

```diff
@@ -48,12 +48,31 @@ Note that we use the double colon (::) to indicate whether a parameter is nested
 | `window` | The accumulation window, such as 1 month, 7 days, 1 year. This can't be used with `grain_to_date`. | Optional  |
 | `grain_to_date` | Sets the accumulation grain, such as `month`, which will accumulate data for one month and then restart at the beginning of the next. This can't be used with `window`. | Optional |
 | `type_params::measure` | A list of measure inputs | Required |
-| `measure:name` | The measure you are referencing. | Optional  |
+| `measure:name` | The name of the measure being referenced. Required if using object syntax for `type_params::measure`.  | Optional  |
 | `measure:fill_nulls_with` | Set the value in your metric definition instead of null (such as zero).| Optional |
 | `measure:join_to_timespine` | Boolean that indicates if the aggregated measure should be joined to the time spine table to fill in missing dates. Default `false`. | Optional |
 
 </VersionBlock>
 
+<Expandable alt_header="Explanation of type_params::measure">
+  
+When reading about `type_params::measure`, you may notice there are different ways of writing it:
```

Comment:
Reading where? We should link to the doc. If we're referencing something here in this block, maybe something like this

```suggestion
The`type_params::measure` configuration can be written in different ways:
```

Reply:
yes in the doc overall!